src/lib/simulator/physics/LennardJones.cpp

```diff
@@ -0,0 +1,52 @@
+#include "LennardJones.h"
+
+#include <array>
+#include <cmath>
+#include <limits>
+
+#include "spdlog/spdlog.h"
+#include "utils/ArrayUtils.h"
+
+namespace simulator::physics {
+/**
+   * @brief Calculates the Lennard Jones force between two particles.
+   *
+   * Computes the Lennard Jones forces based on the positions of two particles.
+   *
+   * @param particle1 The first particle.
+   * @param particle2 The second particle.
+   * @return std::array<double, 3> The calculated force vector.
+   */
+auto LennardJones::calculate_force(const Particle &particle1, const Particle &particle2) -> std::array<double, 3> {
+  const auto epsilon = 5;
+  const auto sigma = 1;
```

Comment:
This can be moved out of the `calculate_force` function to avoid re declaring again and again. The values remain same throughout the simulation.